PWGGA/GammaConv/AliAnalysisTaskOmegaToPiZeroGamma.cxx

```diff
@@ -1294,6 +1347,8 @@ void AliAnalysisTaskOmegaToPiZeroGamma::UserCreateOutputObjects(){
           fHistoTrueOmegaAlphaPt[iCut]->Sumw2();
           fHistoTrueOmegaEtaPhi[iCut]->Sumw2();
           fHistoTrueOmegaPtPi0Pt[iCut]->Sumw2();
+          fHistoTrueOmegaRestGammaCosAnglePt[iCut]->Sumw2();
+          fHistoTrueOmegaRestPi0CosAnglePt[iCut]->Sumw2();
```

Comment:
Should there be a sumw2 for fHistoTrueDalitzPlot as well?

Reply:
Yes there should, thanks for pointing out.